Lab2/Snake/snake.js

```diff
@@ -579,7 +590,7 @@ const game = {
     keyDown: function (key) {
         /* TODO : respond when game-related keys are pushed down */
         if (this.isPaused) {
-            if (key === "") {
+            if (key === " ") {
```

Comment:
The empty string and the string containing one space character are different things!